Tests/Common/Securities/BuyingPowerModelTests.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
```

Comment:
nit - missing license and unrequired imports

`class BuyingPowerModelTests` -> `public class BuyingPowerModelTests`, missing testFixture